### ISSUE TITLE
Accessibility: Contrast of the green color #6ad541

### ISSUE THREAD

**Issue by etu** [NixOS/nixos-homepage#574]
I find the contrast of `#6ad541` when it's text especially on white background a bit hard to read. It's especially noticeable on this page: https://nixos.org/governance.html

![image](https://user-images.githubusercontent.com/461970/94162179-b9881f00-fe86-11ea-8649-befe4acc3696.png)

The contrast for this green text on white is 1.87. For large text to be accessible and easy to read the recommended contrast should be 3 or even 4.5 for big text (which is the case here). (Link to reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast )

Just changing this color to the darker end of the gradient, for example `#3c861f` gave me a contrast of 4.54 and the following result that I find way easier to read.

![image](https://user-images.githubusercontent.com/461970/94162703-4501b000-fe87-11ea-9546-edb6ff242261.png)

I also find the green color to make the page look a bit busy, we have a very nice white theme with shades of blue going on. Then we have stabs of green pointing out here and there. In general I really like the update, except the green.

# Some actual suggestions for replacing the green color

I did some experimentation in my devtools locally and tried to have `#4a53d2` as color for certain elements on the mentioned page above, and most places on the startpage (not the places with dark background).

![image](https://user-images.githubusercontent.com/461970/94164718-ac206400-fe89-11ea-9e5a-7a7d1e09a187.png)

Same color applied a bit down on startpage:
![image](https://user-images.githubusercontent.com/461970/94164769-bf333400-fe89-11ea-88fb-c7513dce8a9e.png)

Then I selected `#aae4fb` for the text and `#7d85f7` for the left button:
![image](https://user-images.githubusercontent.com/461970/94164875-dd992f80-fe89-11ea-9b8f-64be7d6e7ade.png)

**Comment by davidak** [comment]
I agree with the contrast issue, but the suggested colors don't really match the design.

**Comment by etu** [comment]
@davidak I'm not a designer, I just played around with the colorpicker :slightly_smiling_face: And with that said, as a non-designer I still think the green doesn't match the selected plaette. But that seems like it will be addressed according to @garbas comment above, hopefully by someone with a better eye for the design than me :slightly_smiling_face:

**Comment by flyfloh** [comment]
I couldn't find any of the green headings on a white background any more. Feel free to re-open if I missed anything.
